spec/unit/shoulda/matchers/active_model/validate_length_of_matcher_spec.rb

```diff
@@ -161,6 +161,40 @@
     end
   end
 
+  context 'an attribute with some type of normalization on it' do
+    subject do
+      define_model(:example, attr: :string) do
+        validates_length_of :attr, is: 4
+        validates_numericality_of :attr
+
+        def attr=(value)
+          value = value.respond_to?(:upcase) ? value.upcase : value
+          write_attribute(:attr, value)
+        end
+      end.new
+    end
+
+    context 'when ignoring_interference_by_writer is not specified' do
+      it 'should raise CouldNotSetAttributeError' do
+        expect {
+          is_expected.to validate_length_of(:attr).is_equal_to(4)
+        }.to raise_error(
+          Shoulda::Matchers::ActiveModel::AllowValueMatcher::CouldNotSetAttributeError
```

Comment:
Line is too long. [86/80]<br>Put a comma after the last parameter of a multiline method call.